{
  "url": "https://api.github.com/repos/facebook/react/issues/7441",
  "repository_url": "https://api.github.com/repos/facebook/react",
  "labels_url": "https://api.github.com/repos/facebook/react/issues/7441/labels{/name}",
  "comments_url": "https://api.github.com/repos/facebook/react/issues/7441/comments",
  "events_url": "https://api.github.com/repos/facebook/react/issues/7441/events",
  "html_url": "https://github.com/facebook/react/issues/7441",
  "id": 169900949,
  "node_id": "MDU6SXNzdWUxNjk5MDA5NDk=",
  "number": 7441,
  "title": "Pass default parameter values in function",
  "user": {
    "login": "nilaybrahmbhatt",
    "id": 6310631,
    "node_id": "MDQ6VXNlcjYzMTA2MzE=",
    "avatar_url": "https://avatars0.githubusercontent.com/u/6310631?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/nilaybrahmbhatt",
    "html_url": "https://github.com/nilaybrahmbhatt",
    "followers_url": "https://api.github.com/users/nilaybrahmbhatt/followers",
    "following_url": "https://api.github.com/users/nilaybrahmbhatt/following{/other_user}",
    "gists_url": "https://api.github.com/users/nilaybrahmbhatt/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/nilaybrahmbhatt/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/nilaybrahmbhatt/subscriptions",
    "organizations_url": "https://api.github.com/users/nilaybrahmbhatt/orgs",
    "repos_url": "https://api.github.com/users/nilaybrahmbhatt/repos",
    "events_url": "https://api.github.com/users/nilaybrahmbhatt/events{/privacy}",
    "received_events_url": "https://api.github.com/users/nilaybrahmbhatt/received_events",
    "type": "User",
    "site_admin": false
  },
  "labels": [

  ],
  "state": "closed",
  "locked": false,
  "assignee": null,
  "assignees": [

  ],
  "milestone": null,
  "comments": 1,
  "created_at": "2016-08-08T11:26:47Z",
  "updated_at": "2016-08-08T11:51:51Z",
  "closed_at": "2016-08-08T11:51:51Z",
  "author_association": "NONE",
  "body": "I got this error after updating react-native and react\n\ni think this is right syntex  but i dont know why this error is coming\n\n```\nthis.list = function(table, callback, wheredata = [],selection = '',extra = '') {\n                                                  ^\n\nSyntaxError: Unexpected token =\n    at exports.runInThisContext (vm.js:53:16)\n    at Module._compile (module.js:387:25)\n    at Object.Module._extensions..js (module.js:422:10)\n    at Module.load (module.js:357:32)\n    at Function.Module._load (module.js:314:12)\n    at Module.require (module.js:367:17)\n    at require (internal/module.js:16:19)\n    at Object.<anonymous> (/Users/apcrat/Documents/redux/delfoo_React/api/model/catalog/cuisine_route.js:7:12)\n    at Module._compile (module.js:413:34)\n    at Object.Module._extensions..js (module.js:422:10)\n```\n",
  "closed_by": {
    "login": "gaearon",
    "id": 810438,
    "node_id": "MDQ6VXNlcjgxMDQzOA==",
    "avatar_url": "https://avatars0.githubusercontent.com/u/810438?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/gaearon",
    "html_url": "https://github.com/gaearon",
    "followers_url": "https://api.github.com/users/gaearon/followers",
    "following_url": "https://api.github.com/users/gaearon/following{/other_user}",
    "gists_url": "https://api.github.com/users/gaearon/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/gaearon/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/gaearon/subscriptions",
    "organizations_url": "https://api.github.com/users/gaearon/orgs",
    "repos_url": "https://api.github.com/users/gaearon/repos",
    "events_url": "https://api.github.com/users/gaearon/events{/privacy}",
    "received_events_url": "https://api.github.com/users/gaearon/received_events",
    "type": "User",
    "site_admin": false
  }
}
